### PR TITLE
linking to the run-build documentation

### PR DESCRIPTION
**Pull request by mw-hrastega**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,31 +5,31 @@ Before you run MATLAB&reg; code and Simulink&reg; models on a [GitHub&reg;-hoste
...
+name: Run MATLAB Build on GitHub-Hosted Runner
 on: [push]
 jobs:
   my-job:
-    name: Run MATLAB Script
+    name: Run MATLAB Build
     runs-on: ubuntu-latest
     steps:
       - name: Check out repository
         uses: actions/checkout@v2
```
Comment (mcafaro) [inline README.md:22]: They've moved to v3
```suggestion
        uses: actions/checkout@v3
```

File: README.md [hunk 2]
```diff
@@ -50,19 +50,41 @@ jobs:
...
+```yaml
+name: Run MATLAB Script
+on: [push]
+jobs:
+  my-job:
+    name: Run MATLAB Script
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
```
Comment (mcafaro) [inline README.md:65]:
```suggestion
        uses: actions/checkout@v3
```

**Review by mcafaro — APPROVED** [review]

**Review by mcafaro — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -5,31 +5,31 @@ Before you run MATLAB&reg; code and Simulink&reg; models on a [GitHub&reg;-hoste
...
     steps:
       - name: Check out repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Set up MATLAB
         uses: matlab-actions/setup-matlab@v1
-      - name: Run script
-        uses: matlab-actions/run-command@v1
+      - name: Run build
+        uses: matlab-actions/run-build@v1
```
Comment (davidbuzinski) [inline README.md:26]: @v0

Maybe run-build should've (or should) be released as v1.0.0, but currently it's v0.1.0

**Review by davidbuzinski — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (mcafaro) [reply README.md:26]: Oh dang. Can we just burn v0 and push up v1?

Comment (davidbuzinski) [reply README.md:26]: Yeah, I think should be fine. Do we want to delete that release or just release a new v1.0.0?

**Review by davidbuzinski — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (mcafaro) [reply README.md:26]: We can keep v0 and just push up v1 on top

Comment (davidbuzinski) [reply README.md:26]: Sounds good. I just bumped to v1. It should be up now

**Pull request merged**